src/inject.js

```diff
@@ -132,20 +132,24 @@ const checkForRepoPage = async () => {
   const ns = document.querySelector('ul.UnderlineNav-body')
   const navElem = document.getElementById(NAV_ELEM_ID)
   const tdElems = document.querySelector('span.github-repo-size-div')
+  
+  // whether 'add file' button is present
+  const viewingDir = document.querySelector('div.file-navigation.mb-3.d-flex.flex-items-start > details.details-overlay.details-reset.position-relative > summary > span.btn.d-none.d-md-flex.flex-items-center') || document.querySelector('div.file-navigation.mb-3.d-flex.flex-items-start > div.d-flex > details > summary > span.btn.d-none.d-md-flex.flex-items-center')
```

Comment:
Can we use distinctive classes here, instead of all classes? It will also safeguard us from micro changes github makes to the pages.

Reply:
Maybe we can detect the presence of rows in the files list box, instead of using the add file button.
This query-selector can be used :-
`const viewingDir = document.querySelector('div.Box-row.Box-row--focus-gray.py-2.mt-0.border-top-0.d-flex.position-relative.js-navigation-item')`

I'll be adding this change in my next commit.